cli/tests/test_git_init.rs

```diff
@@ -234,6 +234,85 @@ fn test_git_init_external_import_trunk(bare: bool) {
     }
 }
 
+#[test_case(false; "full")]
+#[test_case(true; "bare")]
+fn test_git_init_external_import_trunk_upstream_takes_precedence(bare: bool) {
```

Comment:
nit: remove `test_case(false|true)`? I don't think the repository type matters here.